data/scenario-test.cfg

```diff
@@ -532,6 +532,71 @@ Xu, Xu, Qxu, Qxu, Ql, Ql, Ql, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Gg, Gg, Gg, Gg, Gg
             vision=15
         [/modify_unit]
     [/event]
+    [event]
+        name=attack_end
+        first_time_only=yes
+
+        [filter_attack]
+            special=chance_to_hit
+        [/filter_attack]
+
+        [message]
+            speaker=unit
+            message="I used a chance_to_hit attack specials."
+        [/message]
+    [/event]
+    [event]
+        name=attack_end
+        first_time_only=yes
+
+        [filter_attack]
+            special_id=magical
+        [/filter_attack]
+
+        [message]
+            speaker=unit
+            message="I used a magical attack."
+        [/message]
+    [/event]
+    [event]
+        name=attack_end
+        first_time_only=yes
+
+        [filter_attack]
+            special_id=chance_to_hit
+        [/filter_attack]
+
+        [message]
+            speaker=unit
+            message="a weapon specials with id=chance_to_hit, serious?"
```

Comment:
Please make the message clearer:

`message="BUG: This message should never be displayed!"`